src/invidious/views/components/channels/channel-information.ecr

```diff
@@ -0,0 +1,151 @@
+                    <li class="pure-menu-item pure-menu-selected">
+                    <a href="/channel/<%= channel.ucid %>/videos" class="pure-menu-link">
+                        <b><%= translate(locale, "Videos") %></b>
+                    </a>
+                    </li>
+                <% else %>
+                    <li class="pure-menu-item pure-menu">
+                    <a href="/channel/<%= channel.ucid %>/videos" class="pure-menu-link">
+                        <%= translate(locale, "Videos") %>
+                    </a>
+                    </li>
+                <% end %>
+            <% end -%>
+
+            <% if content_type == 2 %>
+                <li class="pure-menu-item pure-menu-selected">
+                    <a class="pure-menu-link" href="/channel/<%= channel.ucid %>/playlists">
+                    <b> <%= translate(locale, "Playlists") %> </b>
+                    </a>
+                </li>
+            <% else %>
+                <li class="pure-menu-item">
+                    <a class="pure-menu-link" href="/channel/<%= channel.ucid %>/playlists">
+                    <%= translate(locale, "Playlists") %>
+                    </a>
+                </li>
+            <% end -%>
+
+            <% if channel.tabs.includes?("community") %>
+                <% if content_type == 3 %>
+                    <li class="pure-menu-item pure-menu-selected">
+                        <a class="pure-menu-link" href="/channel/<%= channel.ucid %>/community">
+                        <b> <%= translate(locale, "Community") %> </b>
+                        </a>
+                    </li>
+                <% else %>
+                    <li class="pure-menu-item">
+                        <a class="pure-menu-link" href="/channel/<%= channel.ucid %>/community">
+                        <%= translate(locale, "Community") %>
+                        </a>
+                    </li>
+                <% end %>
+            <% end -%>
+
+            <% if content_type == 5 %>
+                <li class="pure-menu-item pure-menu-selected">
+                    <a class="pure-menu-link" href="/channel/<%= channel.ucid %>/about">
+                    <b> <%= translate(locale, "About") %> </b>
+                    </a>
+                </li>
+            <% else %>
+                <li class="pure-menu-item">
+                    <a class="pure-menu-link" href="/channel/<%= channel.ucid %>/about">
+                    <%= translate(locale, "About") %>
+                    </a>
+                </li>
+            <% end -%>
+
+            <% if !channel.auto_generated %>
+                <li class="pure-menu-item" style="margin: 0 0 0 0.5em">
+                    <div class="searchbar">
+                        <form class="pure-form" action="/search" method="get">
+                            <fieldset>
+                                <button class="search-button" style="display: inline;"><i class="icon ion-md-search"></i></button>
```

Comment:
This should contain either a `title` or an `aria-label` attribute to describe what this input/button are for ("search on this channel")